docs/_brainstorming/microsoft-agent-framework-integration.md

```
@@ -2,7 +2,7 @@
 
 **Date**: 2025-12-19
 **Status**: BRAINSTORMING
-**Reference**: `_reference/agent-framework/`
+**Reference**: https://github.com/microsoft/agent-framework
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)